src/components/skybox.js

```diff
@@ -280,6 +288,20 @@ AFRAME.registerComponent("skybox", {
 
       this.el.object3D.matrixNeedsUpdate = true;
     }
+
+    this.updateEnvironmentMap();
+  },
+
+  updateEnvironmentMap() {
+    this.skyScene.add(this.sky);
```

Comment:
Shouldn't this just get added once in init? As it is you would end up adding it to the scene again each time any skymap property changes.

Reply:
add also removes it from the main scene https://github.com/mrdoob/three.js/blob/master/src/core/Object3D.js#L384 I create the skyScene so I can just render the skybox with the cube camera.